src/msw/webview_edge.cpp

```diff
@@ -268,6 +268,19 @@ class wxWebViewConfigurationImplEdge : public wxWebViewConfigurationImpl
 #endif
     }
 
+    bool SetProxy(const wxString& proxy)
+    {
+#ifdef __VISUALC__
+        m_webViewEnvironmentOptions->put_AdditionalBrowserArguments(
```

Comment:
it can not call on MinGW?

Reply:
I am not sure about Edge support status with MinGW and I didn't have time to test it myself, so I've just copied the same test that was already present above. If you (or anybody else) uses Edge with MinGW and can check whether this `put_` call (as well as the one above) works, it would be nice!

Reply:
The problem isn't the call as such, but the EnvironmentOptions object.
It's defined in an Edge SDK header but it's implementation uses a VisualC only COM implementation templates.